lib/galaxy/model/__init__.py

```diff
@@ -3592,6 +3594,29 @@ def to_dict(self, view='collection'):
             rval['metadata_' + name] = val
         return rval
 
+    def updateParentFolderUpdateTimes(self):
+        # sets the update_time for all continaing folders up the tree
+        ldda = self
+        db_session = object_session(ldda)
```

Comment:
try adding
```
db_session.expire_all()
db_session.flush()
```

here to make sure that something in memory is not overriding the objects. Idea of @jmchilton

Reply:
no dice, says Brad